tests/TestDataService.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect } from '../src/index';
+
+test('Data Service', async ({
+    TestDataService,
+    DefaultSalesChannel,
+}) => {
+
+    const category = await TestDataService.createCategory(DefaultSalesChannel.salesChannel.navigationCategoryId, { name: 'Custom Category' });
```

Comment:
Could we use the `DefaultSalesChannel.salesChannel.navigationCategoryId` by default and maybe also move it into props? 
Something like `namePrefix` or `nameSuffix` would also be useful, the name could then be build by combining it with the unique id.

Reply:
I added more default settings to the constructor and made more parameters optional.

I also added a general default config for `namePrefix` and `nameSuffix` and added it to all entity names.